src/Persistent/Api.hs

```diff
@@ -65,6 +65,7 @@ $(makeAcidic ''AulaData
     , 'addIdeaVoteResult
     , 'revokeWinnerStatus
     , 'addLikeToIdea
+    , 'removeLikeFromIdea
```

Comment:
Why is it not `OnIdea` any more here?  Just to resolve the name conflict?  Or because it feels the same?